README.md

```diff
@@ -94,114 +94,135 @@ Snap deploys as a binary, which makes requirements quite simple. We've tested on
 
 ### Installation
 
-You can get the pre-built binaries for your OS and architecture at snap's [GitHub Releases](https://github.com/intelsdi-x/snap/releases) page. This isn't the comprehensive list of plugins, but they will help you get started. Right now, snap only supports Linux and OS X (Darwin).
-
-#### Building Snap
-If you're looking for the bleeding edge of snap, you can build it by getting the `master` branch using `go get github.com/intelsdi-x/snap`. Otherwise you can just use the binaries. To build snap from source, you will need [Golang >= 1.5](https://golang.org) and [GNU Make](https://www.gnu.org/software/make/). For more on building snap check out [BUILD_AND_TEST.md](docs/BUILD_AND_TEST.md).
+You can get the pre-built binaries for your OS and architecture at snap's [GitHub Releases](https://github.com/intelsdi-x/snap/releases) page. This isn't the comprehensive list of plugins. Right now, snap only supports Linux and OS X (Darwin).
 
 ### Running Snap
+We're going to assume you downloaded the latest packaged release of snap and its plugins from here on. If you prefer to build from source, follow the steps in [BUILD_AND_TEST.md](docs/BUILD_AND_TEST.md).
 
-Start a standalone snap agent (snapd):
-
-If you cloned down the repository and aren't using just downloaded binaries, you can set the following. Also, follow the steps in [BUILD_AND_TEST.md](docs/BUILD_AND_TEST.md):
+Untar the version of Snap you downloaded and move its binaries, `snapd` and `snapctl`, to a place in your path. Here is an example of doing so (on Linux):
 ```
-$ export SNAP_PATH=<snapDirectoryPath>/build
+$ tar -xvf snap-v0.13.0-beta-linux-amd64.tar
```

Comment:
Maybe it would be nice to have a `curl` command to save it as `snap-latest.tar` so it is easier for people to follow. Not necessary, but would be nice